presto-docs/src/main/sphinx/release/release-0.280.rst

```diff
@@ -0,0 +1,36 @@
+=============
+Release 0.280
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a race condition, where if a query was submitted before the resource group configuration had been loaded by the Presto server, all queries to the cluster would fail with a ``NullPointerException``. 
+* Fix a bug in the ``GatherAndMergeWindows`` optimization would produces invalid plans when the output of a window function was used in the frame definition of another window function.
+* Fix output of :func:`find_first` function for ``NULL`` Now it will throw exception if the found matched value is NULL.
+* Fix a bug where duplicate items in ``UNNEST`` would lead to query compilation failure.
+* Improve the column access checks for :func:transform and :func:cardinality functions, such that only the required subfields are checked.
```

Comment:
Why :func:transform and :func:cardinality are without quotation marks? Can they resolve to the correct URL after compilation?